reladomo/src/main/java/com/gs/fw/common/mithra/database/AlertHandler

```diff
@@ -0,0 +1,6 @@
+package com.gs.fw.common.mithra.database;
+
+public interface AlertHandler
+{
+    void raiseAlert(String username, String database, String source, String error);
```

Comment:
what's the point of this API? We use standard logging frameworks so any alerting concern can be implemented in a uniform, standard fashion.